scu_builders.py

```diff
@@ -327,7 +330,7 @@ def generate_scu_files(max_includes_per_scu):
     process_folder(["modules/openxr"], ["register_types"])
     process_folder(["modules/openxr/action_map"])
     process_folder(["modules/openxr/editor"])
-    # process_folder(["modules/openxr/extensions"])  # Sensitive include order for platform code.
+    process_folder(["modules/openxr/extensions"])  # Sensitive include order for platform code.
```

Comment:
Might need adjustment I'll see why this was commented out. If there's an include order issue on e.g. windows we may be able to just exclude a file instead.

Reply:
Android build now fixed with a couple of file exclusions.

We could alternatively just add the python equivalent of `#ifndef ANDROID`, but the exclusions should work unless we have any problems on other platforms.